$NetBSD: patch-aa,v 1.6 2010/11/23 20:37:32 wiz Exp $

Fix compilation problem on NetBSD:
  CC     libshm_transmitter_la-fs-shm-stream-transmitter.lo
In file included from fs-shm-transmitter.c:56:
/usr/include/netinet/ip.h:67: error: field 'ip_src' has incomplete type
/usr/include/netinet/ip.h:67: error: field 'ip_dst' has incomplete type
/usr/include/netinet/ip.h:149: error: expected specifier-qualifier-list before 'n_time'
/usr/include/netinet/ip.h:187: error: field 'ippseudo_src' has incomplete type
/usr/include/netinet/ip.h:188: error: field 'ippseudo_dst' has incomplete type

Reported upstream as
https://bugs.freedesktop.org/show_bug.cgi?id=31873

--- transmitters/shm/fs-shm-transmitter.c.orig	2010-09-21 21:22:33.000000000 +0000
+++ transmitters/shm/fs-shm-transmitter.c
@@ -53,7 +53,6 @@
 #else /*G_OS_WIN32*/
 # include <netdb.h>
 # include <sys/socket.h>
-# include <netinet/ip.h>
 # include <arpa/inet.h>
 #endif /*G_OS_WIN32*/
 
